packages/datatrak-web/src/components/Tile.tsx

```diff
@@ -76,6 +93,35 @@ const LoadingContainer = styled.div`
   }
 `;
 
+const ButtonContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```

Comment:
Far from critical, there are a couple places in this file where logical properties would be more appropriate

```suggestion
  inline-size: 100%;
```